apps/ui/src/context/config.tsx

```diff
@@ -1,8 +1,11 @@
-import { ReactNode, createContext, useCallback, useState } from "react";
+import { ReactNode, createContext, useState } from "react";
 
 export interface ConfigContextProps {
   backendURL: string;
   setBackendURL: (backendURL: string) => void;
+  handleEnableRoute: (type: string) => void;
+  handleDisableRoute: (type: string) => void;
+  routes: string[];
```

Comment:
I find the concept of 'routes' confusing here, I wonder if just having two boolean values: 

1. `isHistoryEnabled`
2. `isFavouritesEnabled`

Would be clearer. @kreshnnikgashi